.github/workflows/tests.yml

```diff
@@ -20,5 +20,9 @@ jobs:
         run: |
           pip install poetry
           poetry install
+      - name: Format with Black
+        run: |
+          poetry add black
```

Comment:
`poetry install` installs everything we have in the `pyproject.toml` right? Do we want to maybe include black in the `pyproject.toml` directly then? Then we dont have to add it every time and can also run it inbetween commits just to format the code from time to time manually which i like to do but maybe im the only one on this so this might be just a taste thing

Reply:
Then also the line length 79. is already custom and doens’t need to be specified when running the cmd `black`